### PR TITLE
MKS-53 Add Movies Endpoint

### PR DESCRIPTION
**Pull request by tann96**

### CODE REVIEW

**Review by Magfiozy — APPROVED** [review]

File: Tetiana_Sulyma/Tetiana_Test/Controllers/MoviesController.cs [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
     [Route("api/movies")]
     [ApiController]
     public class MoviesController : ControllerBase
     {
-        // const connectionString = 
-        // GET api/movies
+          // GET api/movies
         [HttpGet]
-        public IEnumerable<string> Get()
+        public ActionResult<IEnumerable<MovieModel>> Get()
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Controllers/MoviesController.cs]: IActionResult

Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Controllers/MoviesController.cs]: I tried, but it show error

**Review by bvzayats — COMMENTED** [review]
There are a lot of mistakes, we will talk later about it.

**Review by bvzayats — COMMENTED** [review]
Look how it implemented in completed and merged PRs and do it

**Review by bvzayats — APPROVED** [review]

**Pull request merged**